wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss

```diff
@@ -243,24 +240,33 @@
     &-heading {
       display: flex;
       gap: rem(12);
-      justify-content: center;
+      justify-content: start;
       align-items: center;
       margin-bottom: rem(24);
-      @extend %text-style-heading-large-strong-caps;
+      @extend %text-style-heading-large-strong;
+      color: $color-accent-primary-blue-100;
+    }
 
-      @include breakpoint('small') {
-        margin-bottom: rem(32);
-      }
+    &-related-posts,
+    &-related-terms {
+      @include breakpoint('medium') {
+        display: inline-block;
 
-      @include breakpoint('large') {
-        margin-bottom: rem(40);
-      }
+        &:nth-of-type(1) {
+          width: calc(100% * 2 / 3);
+        }
 
-      .icon {
-        height: 0.5em;
-        font-size: 2em;
+        &:nth-of-type(2) {
+          width: calc(100% / 3);
+        }
       }
     }
+
+    &-label {
+      @extend %text-style-label-small-strong;
+      color: $color-black-165;
+      text-transform: uppercase;
```

Comment:
This should be removed and the `text--caps` class should be added to the elements.